CHANGELOG.md

```diff
@@ -10,26 +10,61 @@ CHANGELOG
 - Add support for Centos 7 ARM.
 - Add support for FSx Lustre DataCompressionType feature.
 - Add validation to prevent using a `cluster_resource_bucket` that is in a different region than the cluster.
+- Make sure slurmd service is not enabled before finalize stage, which will prevent user from unintentionally making compute node available in post-install process.
```

Comment:
Can we reword it in something that makes more clear the enhancement in terms of user' experience of a functionality. If it's minor let's remove it.

Reply:
Removed, the change can be found in cookbook changelog